test/edu/stanford/dlss/wowza/TestSulWowza.java

```diff
@@ -738,6 +738,45 @@ public void getVerifyStacksTokenUrl_validUrl()
     }
 
     @Test
+    public void verifyTokenAgainstStacksService_filenameNeedsEncoding()
+    {
+        WMSProperties mockProperties = mock(WMSProperties.class);
+        when(mockProperties.getPropertyStr(anyString(), anyString())).thenReturn(SulWowza.DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL);
+        IApplicationInstance appInstanceMock = mock(IApplicationInstance.class);
+        when(appInstanceMock.getProperties()).thenReturn(mockProperties);
+        testModule.onAppStart(appInstanceMock);
+        String druid = "oo000oo0000";
+        String filename = "{([special-chars])}: ü@?;=&#$%20^*.|-_+!,~'\"`";
+        String userIp = "0.0.0.0";
+        String expPath = "/media/" + druid + "/" + SulWowza.urlEncode(filename) + "/verify_token";
```

Comment:
if the test is explicitly calling urlEncode(), then shouldn't there be tests with mocks verifying that verifytokenAgainstStacksService calls urlEncode()  method for stacks token and for filename?   Or is this sufficient?